docs/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,101 @@
+# Deploy Worker and UDF Binaries General Instruction
+
+This documentation will describe how to deploy Worker and UDF (User Defined Function) binaries, which Environment Variable to set up and some commonly used options when launching applications with `spark-submit`.
+
+## Table of Contents
+- [Configurations](#configurations)
+  - [Environment Variables](#1-environment-variables)
+  - [Parameter Options](#2-parameter-options)
+- [Frequently Asked Questions](#faq)
+
+## Configurations
+
+### 1. Environment Variables
+Please see the commonly used environment variables as below when deploying worker and writing udfs. 
```

Comment:
Can we change this to something like: Please make sure the following (relevant) environment variables are set accordingly when deploying worker binaries....?

Reply:
How about this?
```suggestion
Please see the commonly used environment variables as below when deploying worker and writing udfs. And make sure these environment variables are set accordingly.
```